exporters/aws-otel-otlp-udp-exporter/build.gradle.kts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+plugins {
+  id("java")
+  id("java-library")
+  id("maven-publish")
+}
+
+group = "software.opentelemetry.exporters.otlp.udp"
+version = "0.0.1"
+
+dependencies {
```

Comment:
Need help from an expert of `aws-observability/aws-otel-java-instrumentation` to determine if all the dependencies are on the right version.